build.gradle

```diff
@@ -1,16 +1,18 @@
 dependencies {
-    api project(':expo-core')
-    api project(':expo-react-native-adapter')
-    api project(':expo-constants-interface')
+    api project(':unimodules-core')
+    api project(':unimodules-react-native-adapter')
+
     api project(':expo-constants')
-    api project(':expo-sensors-interface')
-    api project(':expo-permissions-interface')
-    api project(':expo-font-interface')
-    api project(':expo-image-loader-interface')
-    api project(':expo-barcode-scanner-interface')
-    api project(':expo-face-detector-interface')
-    api project(':expo-file-system-interface')
-    api project(':expo-camera-interface')
     api project(':expo-file-system')
     api project(':expo-permissions')
+
+    api project(':unimodules-barcode-scanner-interface')
+    api project(':unimodules-camera-interface')
+    api project(':unimodules-constants-interface')
+    api project(':unimodules-face-detector-interface')
+    api project(':unimodules-file-system-interface')
+    api project(':unimodules-font-interface')
+    api project(':unimodules-image-loader-interface')
+    api project(':unimodules-permissions-interface')
+    api project(':unimodules-sensors-interface')
```

Comment:
Since we renamed `@unimodules/core` to `react-native-unimodules`, there is probably no much sense to keep these `build.gradle`, `settings.gradle` files for backwards compatibility, as people will need to install different package anyway. What do you think? 🤔